daemon/daemon.go

```diff
@@ -213,6 +204,21 @@ func (daemon *Daemon) restore() error {
 					logrus.Errorf("Failed to restore %s with containerd: %s", c.ID, err)
 					return
 				}
+
+				// we call Mount and then Unmount to get BaseFs of the container
+				if err := daemon.Mount(c); err != nil {
```

Comment:
@cpuguy83 It shouldn't. We call it all the time for cp actions for example. It just increases the refcount.